test/protocol/data/memcache/check_memcache.c

```diff
@@ -310,6 +310,77 @@ START_TEST(test_set)
 }
 END_TEST
 
+START_TEST(test_partial_value)
+{
+#define SERIALIZED_1 "set foo 0 0 6\r\nXYZ"
+#define SERIALIZED_2 "xyz"
+#define SERIALIZED_3 "\r\n"
+#define KEY "foo"
+#define VAL1 "XYZ"
+#define VAL2 "xyz"
+
+    int ret;
+    struct bstring key = str2bstr(KEY);
+    struct bstring val1 = str2bstr(VAL1);
+    struct bstring val2 = str2bstr(VAL2);
```

Comment:
I prefer if `VAL1` and `VAL2` were not the same length as it can hide some arithmetic bug like subtracting remaining number of bytes instead of received bytes